blockprod/src/detail/mod.rs

```diff
@@ -98,10 +98,7 @@ impl BlockProduction {
     }
 
     pub async fn stop_all_jobs(&mut self) -> Result<usize, BlockProductionError> {
-        self.job_manager_handle
-            .stop_all_jobs()
-            .await
-            .map_err(BlockProductionError::JobManagerError)
+        self.job_manager_handle.stop_all_jobs().await.map_err(BlockProductionError::JobManagerError)
```

Comment:
> The reason why IMO we should consider this is to reduce vertical space waste, which should improve code readability.

I know I'm probably in the minority, but I somewhat disagree that moving things to longer lines, and even to one-liners helps readability - and that's coming from a Perl one-line golfer 😆

There's a concept called Saliency [1] which suggests it's better to maximise information in one area rather than spread them to the periphery e.g in the above code, your eye can see the whole modification to `self` because each operation has an obvious "dot" at the start of each line vs the one-liner where your eye has to carefully visually scan (which is easier to miss things).

Anothe thing going for having multiple things over multiple lines is that it makes diffing code a lot easier to read e.g if you were to remove the `await` above, in a multi-line diff it would be obvious at a glance vs having to scan the single line looking for a change.

[1] https://jov.arvojournals.org/article.aspx?articleid=2193531